main.py

```diff
@@ -1,27 +1,47 @@
 """UI Server, Connects all Components."""
-
+import os
+from file_adapter import FileAdapter
 from flask import Flask
 from flask import make_response
 from flask_restful import Api
+import resources
+from resources import Releases
 from resources import Resources
-
+from airflow_connector import AirflowDB
 
 # creating the Flask application
 APP = Flask(__name__)
 API = Api(APP)
+resources = Resources()
+
 
-REST = Resources()
+adapter = FileAdapter('fake_data/fake_release_data.json', 'fake_data/fake_task_data.json')
+cloudsql_unix_socket = os.path.join('/cloudsql', 'istio-release-ui:us-central1:prod-airflow-snapshot-sandbox')
```

Comment:
Use flags https://docs.python.org/3.3/library/argparse.html

Reply:
I still feel that creating the db connection here, and passing it down allows us to test the core more easily.

Reply:
It might be better for testing, but creating it in the adapter allows for me to easily reconnect if the connection is lost, without raising an error and cancelling a query

Reply:
Additionally, I don't believe that App Engine will be able to make use of these flags, so I assume that you want me to make the current values default?